Nifty/Classes/NTDisplay/Utilities/NTTimer.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  NTTimer.swift
+//  CwlUtils
+//
+//  Created by Naveen Chaudhary on 04/12/17.
+//
+
+import JavaScriptCore
+
+@objc public protocol NTTimerNodeExports: JSExport {
+    static func create() -> NTTimer
```

Comment:
We can have 2 apis. 1 for creating timer with an interval and other for creating and scheduling with an interval. Kind of like what NSTimer does.
Lets discuss this offline.